### PR TITLE
setup

### PR DESCRIPTION
**Pull request by sanchaymittal**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,6 @@
         "modules/contracts/deployments"
       ],
       "@connext/nxtp-subgraph": ["modules/subgraph/generated"],
-      "@connext/nxtp-sdk": ["modules/sdk/src"],
```
Comment (jakekidd) [inline tsconfig.json:19]: ... have there been two of these this whole time?

**Review by jakekidd — APPROVED** [review]
lgtm

File: tsconfig.json (hunk 1 above)
Comment (sanchaymittal) [reply tsconfig.json:19]: yeah

Comment (jakekidd) [reply tsconfig.json:19]: lol

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**